zone/parcels.cpp

```diff
@@ -149,32 +154,37 @@ void Client::SendParcel(Parcel_Struct parcel_in)
 			uint32 packet_size = ss.str().length();
 			auto   out         = new EQApplicationPacket(OP_ItemPacket, packet_size);
 			if (out->size != packet_size) {
-				LogError("Attempted to send a parcel packet of mismatched size {} with a buffer size of {}.",
-						 out->Size(), packet_size);
+				LogError(
+					"Attempted to send a parcel packet of mismatched size {} with a buffer size of {}.",
+					out->Size(),
+					packet_size
+				);
 				return;
 			}
+
 			memcpy(out->pBuffer, ss.str().data(), out->size);
 			FastQueuePacket(&out);
 
 			ss.str("");
 			ss.clear();
 
-			parcels.emplace(parcel.slot_id, parcel);
+			m_parcels.emplace(parcel.slot_id, parcel);
 		}
 	}
 }
 
 void Client::DoParcelCancel()
 {
-	if (parcel_platinum || parcel_gold || parcel_silver || parcel_copper) {
-		m_pp.platinum 	+= parcel_platinum;
-		m_pp.gold 		+= parcel_gold;
-		m_pp.silver 	+= parcel_silver;
-		m_pp.copper 	+= parcel_copper;
-		parcel_platinum = 0;
-		parcel_gold     = 0;
-		parcel_silver   = 0;
-		parcel_copper   = 0;
+	if (m_parcel_platinum || m_parcel_gold || m_parcel_silver ||
+		m_parcel_copper) {
+		m_pp.platinum += m_parcel_platinum;
+		m_pp.gold += m_parcel_gold;
+		m_pp.silver += m_parcel_silver;
+		m_pp.copper += m_parcel_copper;
+		m_parcel_platinum = 0;
+		m_parcel_gold     = 0;
+		m_parcel_silver   = 0;
+		m_parcel_copper   = 0;
```

Comment:
Formatting.
```cpp
if (
	m_parcel_platinum ||
	m_parcel_gold ||
	m_parcel_silver ||
	m_parcel_copper
) {
	m_pp.platinum += m_parcel_platinum;
	m_pp.gold     += m_parcel_gold;
	m_pp.silver   += m_parcel_silver;
	m_pp.copper   += m_parcel_copper;

	m_parcel_platinum = 0;
	m_parcel_gold     = 0;
	m_parcel_silver   = 0;
	m_parcel_copper   = 0;
```

Reply:
Made the changes though the formatter does not do this for me.  I must have some setting incorrect.